Automattic/pocket-casts-ios
# Bookmarks: Adds the tabs to the episode detail view

| 🛫 Depends on: https://github.com/Automattic/pocket-casts-ios/pull/1013 | 🎨 Design: [Figma](https://www.figma.com/file/Io7PCz1H1hmOgEE9eljoYW/Bookmarks?type=design&node-id=2036-47334&mode=dev) |
|:---:|:---:|

This adds the Details / Bookmarks tabs to the episode detail view. 

Note: There are some clipping issues on smaller screens that I will address in a future PR.

## Demo Video

https://github.com/Automattic/pocket-casts-ios/assets/793774/1b9c99b1-94b5-47c3-a15c-951d5a50ccfe


## To test

1. Launch the app
2. Enable the Bookmarks feature flag
3. Go to the Podcasts tab
4. Open a podcast that has bookmarks
5. Tap on an episode that has bookmarks
6. ✅ Verify you see the Details / Bookmarks tab
7. Swipe to the bookmarks tab
8. ✅ Verify the tab updates with the current page
9. ✅ Verify the right buttons update to show the more button ( ... )
10. Tap the More button
11. ✅ Verify you see the options picker, and the options work
12. Tap the Details tab
13. ✅ Verify you are brought to the details view
14. ✅ Verify the right buttons update to the toggle star / share items
15. ✅ Verify they work
16. Tap the Bookmarks tab
17. ✅ Verify you are brought to the bookmarks view

## Checklist
